src/Dfe.PlanTech.Application/Content/Queries/GetSubTopicRecommendationFromContentfulQuery.cs

```diff
@@ -0,0 +1,17 @@
+using Dfe.PlanTech.Application.Persistence.Interfaces;
+using Dfe.PlanTech.Domain.Content.Queries;
+using Dfe.PlanTech.Domain.Questionnaire.Models;
+
+namespace Dfe.PlanTech.Application.Content.Queries;
+
+public class GetSubTopicRecommendationFromContentfulQuery(IContentRepository repository) : IGetSubTopicRecommendation
+{
+    private readonly IContentRepository _repository = repository;
+
+    public async Task<SubTopicRecommendation> GetSubTopicRecommendation(Section subTopic, CancellationToken cancellationToken = default)
+    {
+        IEnumerable<SubTopicRecommendation> subTopicRecommendations = await _repository.GetEntities<SubTopicRecommendation>(cancellationToken);
```

Comment:
This will end up retrieving every single subtopic recommendation from Contentful, and then querying on our server side to find the matching one.

We can query by ID directly, and IMO should do that instead as this is wasteful.